doc/source/index.rst

```diff
@@ -15,4 +15,5 @@ found at `in-toto.io <https://in-toto.io>`_.
    api
    model
    layout-creation-example
+   exclude-patterns <Will provide link when available>
```

Comment:
We don't need a link unless it's an external document. Take a look at the others above vs the demo which is an external link. :)